wee_slack.py

```diff
@@ -683,7 +688,7 @@ def buffer_prnt(self, user='unknown_user', message='no message', time=0):
                 else:
                     name = user
                     self.last_active_user = None
-                name = nick_prefix + name + nick_suffix
+                name = nick_prefix_color + nick_prefix + name + nick_suffix_color + nick_suffix + w.color("reset")
```

Comment:
09:30 <Mithrandir_> if you could just fix up using --amend and force-push that's better for me.  
    09:30 <Mithrandir_> since it means I can just mash the merge button
In case someone was wondering why the commit mentioned here dissapeared ;)